library/core/src/cell.rs

```diff
@@ -1905,8 +1890,8 @@ impl<T: ?Sized + fmt::Display> fmt::Display for RefMut<'_, T> {
 ///
 /// ```rust
 /// #![forbid(unsafe_code)] // with exclusive accesses,
-///                         // `UnsafeCell` is a transparent no-op wrapper,
-///                         // so no need for `unsafe` here.
```

Comment:
this formatting may be somewhat questionable, but the rustfmt output is no better